doc/whats_new/upcoming_changes/sklearn.metrics/31671.fix.rst

```diff
@@ -0,0 +1,8 @@
+- :func:`sklearn.metrics.d2_pinball_score` and `d2_absolute_error_score` now always
+  use the `"averaged_inverted_cdf"` quantile interpolation method, both with and
```

Comment:
```suggestion
  use the `"averaged_inverted_cdf"` quantile calculation method, both with and
```

I think since this is a discrete method, no interpolation is happening..?